copy/pq.go

```diff
@@ -87,13 +87,7 @@ func (cb *CopyWithPq) CopyOneBatchCustomImpl(ctx context.Context, srcTableColumn
 	defer srcConnTxn.Commit(ctx)
 
 	// See if we should load the commit with an existing transaction snapshot id.
-	if len(transactionSnapshotId) > 0 {
-		_, err = srcConnTxn.Exec(ctx, fmt.Sprintf("SET TRANSACTION SNAPSHOT '%s'", transactionSnapshotId))
-		if err != nil {
-			panic(err)
```

Comment:
was running into `postgres cannot import a snapshot from a different database` error, removing this block works for now.

Reply:
Let's track this down. We need to set the transaction snapshot so we're copying data from that point in time. Maybe we're breaking the transaction somewhere?